File: src/hello_world/src/helloworld.py
import rospy
from std_msgs.msg import String

def main():
    rospy.init_node('helloworld', anonymous=True)
    pub = rospy.Publisher('abcsdfj', String, queue_size=10)
    rospy.loginfo("Hello, world!")

    rate = rospy.Rate(10)  # 10Hz
    while not rospy.is_shutdown():
        rospy.loginfo("Hello, world!")
        msg = String()
        msg.data = "Hello, ros!"
        pub.publish(msg)
        rate.sleep()

if __name__ == '__main__':
    try:
        main()
    except rospy.ROSInterruptException:
        pass

File: src/hello_world/src/helloworld.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>
int main(int argc, char **argv)
{
  ros::init(argc, argv, "helloworld");
  printf("Hello, world!\n");
  ros::NodeHandle nh;
  ros::Publisher pub = nh.advertise<std_msgs::String>("abcsdfj", 10);
  while(ros::ok())
  {
    printf("Hello, world!\n");
    std_msgs::String msg;
    msg.data = "Hello,ros!";
    pub.publish(msg);
  }
  return 0;
}